third_party/move/move-compiler-v2/transactional-tests/tests/tests.rs

```diff
@@ -44,7 +44,7 @@ const TEST_CONFIGS: &[TestConfig] = &[
             (Experiment::OPTIMIZE_WAITING_FOR_COMPARE_TESTS, true),
             (Experiment::ACQUIRES_CHECK, false),
         ],
-        language_version: LanguageVersion::V2_0,
+        language_version: LanguageVersion::V2_1,
```

Comment:
Why are we bumping the version in all of these tests?  What does V2_1 mean exactly?

Reply:
Copied from `metadata.rs`
```
// Language Version

/// Represents a language version.
///
/// The versioning scheme is `major.minor`, where for `major = 1` we do not
/// distinguish a minor version. This we have versions `1, 2.0, 2.1, .., 3.0, 3.1, ..`.
/// Typically, a major version change is given with an addition of larger new language
/// features. There are no breaking changes expected with major version changes,
/// however, there maybe some isolated exceptions.
#[derive(Clone, Copy, Debug, Serialize, Deserialize, Eq, PartialEq, PartialOrd, Ord)]
pub enum LanguageVersion {
    /// The version of Move at around the genesis of the Aptos network end
    /// of '22. This is the original Diem Move plus the extension of inline
    /// functions with lambda parameters, as well as a simple form of `for`
    /// loops.
    V1,
    /// The 2.0 version of Move.
    V2_0,
    /// The currently unstable 2.1 version of Move
    V2_1,
}

```